packages/create-sitecore-jss/src/templates/nextjs-sxa/src/Layout.tsx

```diff
@@ -10,12 +10,13 @@ import {
   Field,
 } from '@sitecore-jss/sitecore-jss-nextjs';
 import Scripts from 'src/Scripts';
+import { SitecorePageProps } from 'lib/page-props';
 
 // Prefix public assets with a public URL to enable compatibility with Sitecore Experience Editor.
 // If you're not supporting the Experience Editor, you can remove this.
 const publicUrl = getPublicUrl();
 
-interface LayoutProps {
+interface LayoutProps extends Pick<SitecorePageProps, 'headLinks'> {
```

Comment:
Importing and extending `SitecorePageProps` just for `headLinks` seems a bit much. Might be simpler to just re-define the prop directly here? Technically, the name could be something else (we just happened to use the same in `[[...path]].tsx`)

Reply:
Makes sense, can be simplified